src/main/java/org/traccar/notificators/NotificatorTelegram.java

```diff
@@ -55,8 +75,26 @@ public void sendSync(long userId, Event event, Position position) {
 
         Message message = new Message();
         message.chatId = chatId;
-        message.text = NotificationFormatter.formatShortMessage(userId, event, position);
+        message.text = NotificationFormatter.formatFullMessage(userId, event, position);
+        executeRequest(urlSendText, message);
+
+        if (position != null) {
+            LocationMessage locationMessage = new LocationMessage();
+            locationMessage.chatId = chatId;
+            locationMessage.latitude = position.getLatitude();
+            locationMessage.longitude = position.getLongitude();
+            locationMessage.bearing = position.getCourse();
+            locationMessage.accuracy = position.getAccuracy();
+            executeRequest(urlSendLocation, locationMessage);
```

Comment:
Is there a way to send location message together with text?

Reply:
It would be great, but according to [their API](https://core.telegram.org/bots/api#sendlocation) it is not really possible.

I tried adding `text` parameter, but to no avail.

Reply:
Should we at least use `reply_to_message_id` to indicate which message this location is related to?